apps/node-fastify/package.json

```
@@ -22,7 +22,8 @@
     "dotenv": "^16.4.7",
     "fastify": "^5.2.0",
     "orb-sync-lib": "*",
-    "pino": "^9.6.0"
+    "pino": "^9.6.0",
+    "@sentry/node": "8.5.0"
```

Latest should be 8.50.0 and let's use caret version to be consistent 
```suggestion
    "@sentry/node": "^8.50.0"
```